src/modules/general/components/Button/button.types.ts

```diff
@@ -1,7 +1,9 @@
-import { ButtonProps as DefaultProps } from '@mui/material';
+import { ButtonProps as DefaultProps, LinkProps } from '@mui/material';
 
 export interface ButtonProps extends DefaultProps {
   customStyle?: string;
   block?: boolean;
   color: 'primary' | 'secondary' | 'error' | 'info' | 'inherit';
+  to?: string;
+  target?: '_blank' | '_top' | '_self' | '_parent';
```

Comment:
MUI button can act like a Link if we define `to` and `target` props. It is used in one of our components and it was working correctly, but since it was not defined in our Button props, was causing ts errors, I just added the props